DataBase.java

```diff
@@ -48,22 +49,27 @@ public class DataBase {
     public DataBase() {
         passengers = new ArrayList<>();
         trains = new ArrayList<>();
-        users = new ArrayList<>(); // Initialize the users list
+        users = new ArrayList<>();
+        admins = new ArrayList<>();
```

Comment:
is separate admins needed. Why not use  role in user?

Reply:
No need. and user role are changed to enum